### ISSUE TITLE
Release a new version with the upstream

### ISSUE THREAD

**Issue by makimoto** [cookpad/expeditor#48]
Hi, we found expeditor v0.7.0 doesn't work with Ruby 3.0.x or higher due to a change in the interface of keyword arguments.

```
irb(main):001:0> Expeditor
=> Expeditor
irb(main):002:0> Expeditor::Service
=> Expeditor::Service
irb(main):003:0> Expeditor::Service.new
/Users/makimoto/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/expeditor-0.7.0/lib/expeditor/rolling_number.rb:10:in `initialize': wrong number of arguments (given 1, expected 0; required keywords: size, per_time) (ArgumentError)
	from /Users/makimoto/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/expeditor-0.7.0/lib/expeditor/service.rb:111:in `new'
	from /Users/makimoto/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/expeditor-0.7.0/lib/expeditor/service.rb:111:in `block in reset_status!'
	from /Users/makimoto/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/expeditor-0.7.0/lib/expeditor/service.rb:110:in `synchronize'
	from /Users/makimoto/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/expeditor-0.7.0/lib/expeditor/service.rb:110:in `reset_status!'
	from /Users/makimoto/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/expeditor-0.7.0/lib/expeditor/service.rb:19:in `initialize'
	from (irb):3:in `new'
	from (irb):3:in `<main>'
	from /Users/makimoto/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/irb-1.3.5/exe/irb:11:in `<top (required)>'
	from /Users/makimoto/.rbenv/versions/3.0.4/bin/irb:23:in `load'
	from /Users/makimoto/.rbenv/versions/3.0.4/bin/irb:23:in `<main>'
```

It was fixed in https://github.com/cookpad/expeditor/pull/45, and I confirmed the upstream works well with Ruby 3.0.4 and 3.1.2.

So, could you kindly release a new one, including the upstream?
Thank you!

**Comment by eagletmt** [comment]
Released v0.7.1 https://rubygems.org/gems/expeditor/versions/0.7.1

**Comment by makimoto** [comment]
Thank you for the prompt action!
